sys/flash_map/src/flash_map.c

```diff
@@ -21,6 +21,7 @@
 #include <string.h>
 #include <assert.h>
 
+#include <os/os.h>
```

Comment:
yes for min, though that could be duplicated so os isnt needed, or put somewhere else